crates/kornia-linalg/src/linalg.rs

```diff
@@ -0,0 +1,487 @@
+// Reference: https://github.com/wi-re/tbtSVD/blob/master/source/SVD.h
+use glam::{Mat3, Quat, Vec3};
+use std::ops::{Index, IndexMut};
+const GAMMA: f32 = 5.828427124;
+const CSTAR: f32 = 0.923879532;
+const SSTAR: f32 = 0.3826834323;
+const SVD_EPSILON: f32 = 1e-6;
+const JACOBI_STEPS: u8 = 12;
+const RSQRT_STEPS: u8 = 4;
+const rsqrt_mut_STEPS: u8 = 6;
+
+/// Standard CPU division.
+fn fdiv(x: f32, y: f32) -> f32 {
+    return x / y;
+}
+
+/// Calculates the reciprocal square root of x using a fast approximation.
+fn rsqrt(x: f32) -> f32 {
```

Comment:
We might want to try https://betterexplained.com/articles/understanding-quakes-fast-inverse-square-root/ which doesn’t have the loops. That could be the cause difference between faer ?

Reply:
Ok

Reply:
Current benchmark after changing the rsqrt:
![Code_Og4rurVpvO](https://github.com/user-attachments/assets/85a4e78c-2e8a-4f09-bff7-98867b37136b)

Reply:
We are getting closer ! Let’s try to find out how to improve a little bit more

Reply:
Also one suggestion while running benchmarks, try to have as less things as possible running in the background like web browser or slack, etc

Reply:
Ok got it